index2.html

```diff
@@ -0,0 +1,230 @@
+            <div class="photo">
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\1.jpg"
+                        alt="Granny gives everyone the finger">
+                    <div>
+                        <p class="text_after_images">Granny gives everyone the finger, and other tips from OFFF
+                            Barcelona</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\2.jpg" alt="Hello world">
+                    <div>
+                        <p class="text_after_images">Hello world, or, in other words, why this blog exists
+                        </p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\3.jpg"
+                        alt="regarding how wework">
+                    <div>
+                        <p class="text_after_images">Here are some things you should know regarding how we work </p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\4.jpg" alt="remote work">
+                    <div>
+                        <p class="text_after_images">Updating list of 50+ sources on distributed teams, remote
+                            work, and
+                            how to make it all work better</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\5.jpg"
+                        alt="A few words about this blog platform">
+                    <div>
+                        <p class="text_after_images">A few words about this blog platform, Ghost, and how this site was
+                            made</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304px" height="176px" src="photos\6.jpg"
+                        alt="modern remote working">
+                    <div>
+                        <p class="text_after_images">How modern remote working tools get along with Old School
+                            Cowboy's methods
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="sign_up">
+            <h2 class="sign_up_caption">Sign up for the newsletter</h2>
+            <p class="sign_up_text">If you want relevant updates occasionally, sign up for the private newsletter. Your
+                email is never shared.</p>
+            <div class="mail">
```

Comment:
I highly recommend to wrap your inputs and buttons with `form` element. it will allow users to submit forms just by pressing `enter`
```suggestion
            <form class="mail">
```